actionpack/lib/system_test_case.rb

```diff
@@ -0,0 +1,57 @@
+require "system_testing/test_helper"
+require "system_testing/driver_adapter"
+
+module Rails
```

Comment:
I explained why I chose this name in the message above and I'm not going to move it around 100 times before we have a consensus on name.